src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -9055,9 +9055,23 @@ private ExpressionSyntax ParseSubExpressionCore(Precedence precedence)
             {
                 var questionToken = this.EatToken();
                 var colonLeft = this.ParsePossibleRefExpression();
-                var colon = this.EatToken(SyntaxKind.ColonToken);
-                var colonRight = this.ParsePossibleRefExpression();
-                leftOperand = _syntaxFactory.ConditionalExpression(leftOperand, questionToken, colonLeft, colon, colonRight);
+                if (this.CurrentToken.Kind == SyntaxKind.EndOfFileToken && this.lexer.FollowedByColon)
```

Comment:
>FollowedByColon [](start = 86, length = 15)

Consider using more specific name. Here is what the comment says below: "We have an **interpolated string** ...". A name "IsInterpolationExpressionFollowedByColon" could be clearer, for example.

Reply:
Name changed to be more specific.

---
In reply to: [152120207](https://github.com/dotnet/roslyn/pull/23265#discussion_r152120207) [](ancestors = 152120207)